### PR TITLE
Add a simple README.md file

### PR DESCRIPTION
**Pull request by AliJavadiGithub**
Adding a README file that was missing for the project.

### CODE REVIEW

**Comment by NLPAli** [comment]
This README is too short. It would be nice to see an example.

**Comment by NLPAli** [comment]
Thanks! Please have a look at the comments

**Comment by NLPAli** [comment]
> Thanks! Please have a look at the comments

**Review by NLPAli — DISMISSED** [review]
Thanks! Please have a look at the comments

**Review by NLPAli — COMMENTED** [review]
d

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName, FirstName" into "FirstName LastName"
```
Comment (NLPAli) [inline README.md]: This sentence is missing a period at the end

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Example
```
Comment (NLPAli) [inline README.md]: Please make this ## so that it's a smaller title

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Calling `rearrange_name("Turing", "Alan")` will return `"Alan Turing"`
```
Comment (NLPAli) [inline README.md]: This is cool, but please add a couple more examples.

**Review by NLPAli — COMMENTED** [review]
Thanks! Please have a look at the comments

File: README.md (hunk 1 above)
Comment (NLPAli) [inline README.md]: This sentence is missing a period at the end

File: README.md (hunk 2 above)
Comment (NLPAli) [inline README.md]: Please make this ## so that it's a smaller title

File: README.md (hunk 3 above)
Comment (NLPAli) [inline README.md]: This is cool, but please add a couple more examples

**Review by NLPAli — COMMENTED** [review]
Thanks! Please have a look at the comments

**Comment by AliJavadiGithub** [comment]
All comments are resolved, please take another look.

**Pull request closed without merging**